app/synchronizationmanager.cpp

```diff
@@ -311,3 +314,9 @@ void SynchronizationManager::onProjectReloadNeededAfterSync( const QString &proj
   }
 
 }
+
+QString SynchronizationManager::getAvailableStorage()
+{
+  double availableStorage = mMerginApi->workspaceInfo()->storageLimit() - mMerginApi->workspaceInfo()->diskUsage();
+  return InputUtils::bytesToHumanSize( availableStorage );
+}
```

Comment:
You might want to put this function into `MerginApi` when the only thing it does is calling its API :)